test/integration/apiserver/tracing/tracing_test.go

```diff
@@ -388,7 +388,7 @@ endpoint: %s`, listener.Addr().String())), os.FileMode(0755)); err != nil {
 							return v.GetStringValue() == "grpc"
 						},
 					},
-					events: []string{"message"},
+					events: []string{},
```

Comment:
Hmmm, this is concerning, didn't expect a change in tracing test.

Reply:
@dashpole can you please review?

Reply:
Maybe we need to pass `--experimental-distributed-tracing-sampling-rate` flag. Leaving to @dashpole what would be prefered solution.

cc @jmhbnz

Reply:
I believe this is due to https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3964.  I believe it needs to be fixed here: https://github.com/kubernetes/kubernetes/blob/695a984f569bec2835bcb3ac8474cebefd020994/staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go#L319 by adding this option:

```golang
otelgrpc.WithMessageEvents(otelgrpc.ReceivedEvents, otelgrpc.SentEvents),

```

It is probably needed in these places as well:

* https://github.com/kubernetes/kubernetes/blob/695a984f569bec2835bcb3ac8474cebefd020994/pkg/kubelet/cri/remote/remote_image.go#L66
* https://github.com/kubernetes/kubernetes/blob/695a984f569bec2835bcb3ac8474cebefd020994/pkg/kubelet/cri/remote/remote_runtime.go#L95

Reply:
+1 to resolving this, the rest of the changes in the update lgtm

Reply:
Adding `otelgrpc.WithMessageEvents(otelgrpc.ReceivedEvents, otelgrpc.SentEvents)` fixed the issue. Thanks @dashpole, added it to places you pointed.